ci/test-cross-image.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2086
+
+set -x
+set -eo pipefail
+
+if [[ -z "${TARGET}" ]]; then
+    export TARGET="aarch64-unknown-linux-gnu"
+fi
+if [[ -z "${CROSS_TARGET_CROSS_IMAGE}" ]]; then
+    CROSS_TARGET_CROSS_IMAGE="ghcr.io/cross-rs/cross:main"
+fi
```

Comment:
This way we use the CI default if present, but have a reasonable target if testing our CI configurations locally.